PL release notes for 792xx (#1564)
harness/developer-hub: release-notes/whats-new.md

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features described in these release notes may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page.
:::
 
## Latest - May 03, 2023, version 792xx

### Harness Platform

> - You will now receive an alert when there are unsaved changes on the default settings page and if you leave it. (PL-32354)

Reviewer: ```suggestion
- You will now receive an alert on the default settings page when there are unsaved changes, or if you leave the page. (PL-32354)
```

Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features described in these release notes may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page.
:::
 
## Latest - May 03, 2023, version 792xx

### Harness Platform

- You will now receive an alert on the default settings page when there are unsaved changes, or if you leave the page. (PL-32354)
  

> - Variables corresponding to pipelines, stages, services, and environments are optional and no longer validated on the run pipeline form. (CDS-64656, ZD-43232)

Reviewer 2: CD ticket

> - The pipeline abort dialog text is now updated to warn that abort will not clean up resources. It suggests marking the stage as failed if cleanup is required. (CDS-67000)

Reviewer 2: CD ticket
